backend/prisma/schema.prisma

```diff
@@ -76,6 +76,7 @@ model Post {
   id         Int          @default(autoincrement()) @id
   title      String
   body       String
+  excerpt    String
```

Comment:
Here's the new `excerpt` field ⬆️

Reply:
Looks good, I'm guessing the excerpt be like the first N characters of the post (maybe non-heading characters)?

Reply:
Yep exactly, that was my thought! Probably first N chars of the actual body content. In theory, the title of the post is on the header/image, so posts will likely start with body copy anyway, but will be good to have a check for that just in case since it's totally possible to do that with the rich text editor!